src/react-components/src/components/01-atoms/Links/LinkHelper.tsx

```diff
@@ -0,0 +1,57 @@
+// Base Link Helper Class
+import * as React from "react";
+
+export interface LinkHelperProps {
+  className: string;
+  icon?: { element: JSX.Element };
```

Comment:
Are there other components that use the ` { element: JSX.Element }` type for an icon? It seems to add an extra layer to create an object for this icon element when it doesn't have any other properties (unless if you expect to add more later?).

Reply:
I don't know if I understand the question -- but, the reason why `icon` needs to passed in here as an element is that we want the anchor tag to wrap both the text and the icon, as per: 
http://themetronome.co/components/mt-18-text-link-w-icon/?fresh=true

Reply:
I meant more like, does `icon` have to be an object? Can it just be `icon?: JSX.element`? In the parent components I see where the object is being created but it doesn't seem necessary.

Reply:
Not a big deal, just curious.

Reply:
Yeah, I don't know why it needed to be its own object... I think maybe in the past it had a key or something.